lib/w_common.dart

```diff
@@ -12,7 +12,13 @@
 // See the License for the specific language governing permissions and
```

Comment:
Do we want to deprecate this library entry point in favor of the `_browser` or `_vm` one?